### ISSUE TITLE
UpdateManager ctor now requires 2 parameters, docs not updated

### ISSUE THREAD

**Issue by RandomEngy** [clowd/Clowd.Squirrel#31]
As of 2.7.34-pre, UpdateManager requires an `applicationName` parameter. I assume this is the --packName/--packId? But the examples in README.md still have one parameter.

**Comment by caesay** [comment]
I made the second parameter required in 2.7.34 because, if left blank, the `UpdateManager` constructor will throw in non-installed apps (eg. running from Visual Studio). 

I have since refactored UpdateManager and [the new constructor](https://github.com/clowd/Clowd.Squirrel/blob/develop/src/Squirrel/UpdateManager.cs#L54) only has a single required parameter again. In the next pre-release you will be able to instantiate UpdateManager with a single parameter and check if the app is currently installed with `UpdateManager.IsInstalledApp` and/or `UpdateManager.CurrentlyInstalledVersion() without it throwing an exception.
